sympy/sets/ordinals.py

```diff
@@ -1,36 +1,67 @@
 from sympy.core import Basic
 import operator
 
+class OrdinalPow(Basic):
+    """
+    Represents ordinal exponential and multiplication terms one of the
+    building blocks of the Ordinal class.
+    In OrdinalPow(a, b) a represents exponent and b represents multiplicity.
+    """
+    def __new__(cls, *terms):
```

Comment:
The parameters are usually called `args`.

Reply:
If this only expects two argument (?), then better simply have `def __new__(cls, a, b)` this would also avoid having to `raise TypeError` as is currently done.